PlayCover.xcodeproj/project.pbxproj

```diff
@@ -72,6 +72,8 @@
 		AA71964B287A0EA800623C15 /* PlayRules.swift in Sources */ = {isa = PBXBuildFile; fileRef = AA71964A287A0EA800623C15 /* PlayRules.swift */; };
 		AA71964D287A35F400623C15 /* com.nexon.bluearchive.yaml in Resources */ = {isa = PBXBuildFile; fileRef = AA71964C287A35F400623C15 /* com.nexon.bluearchive.yaml */; };
 		AA818CB5287ABEC3000BEE9D /* Yams in Frameworks */ = {isa = PBXBuildFile; productRef = AA818CB4287ABEC3000BEE9D /* Yams */; };
+		AACF11A42B31769C008BAA81 /* UninstallVM.swift in Sources */ = {isa = PBXBuildFile; fileRef = AACF11A32B31769C008BAA81 /* UninstallVM.swift */; };
+		AACF11A62B3176B2008BAA81 /* ClearCacheModalView.swift in Sources */ = {isa = PBXBuildFile; fileRef = AACF11A52B3176B2008BAA81 /* ClearCacheModalView.swift */; };
```

Comment:
Typo, should be `ClearCacheModelView` (change "a" in "Modal" to "e"). The struct name has the same typo.

Reply:
"Modal" is not a typo, which refers to a dialog box like an alert, popover, and sheet. 
Check this out: [https://developer.apple.com/documentation/swiftui/modal-presentations#Overview](url)